### PR TITLE
Added changelog entries that we missed

### PR DESCRIPTION
**Pull request by mauricioszabo**
Just what it says :D

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,21 @@
 
 ## [Unreleased]
 
+- Fixed a bug where MacOSX menus like "Open" don't do anything
```
Comment (Daeraxa) [inline CHANGELOG.md:10]:
```suggestion
- Fixed a bug where macOS menus like "Open" don't do anything
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,21 @@
...
+- Fixed a bug where the editor refused to open with the message "GPU process
+isn't usable. Goodbye" (https://github.com/pulsar-edit/pulsar/issues/233)
+- Fixed logo artifacts on Linux
+- Fixed commands like `--version`, `--package` or `--help` did not show outputs
+- Fixed additional flags not being sent to `--package`
+- Small improvement on the binary size
+- Fixed "install command line tools" on Mac and Windows
+- Cached queries for featured packages (featured packages will load faster, and
+less errors on the settings-view regarding package info)
+- Added warning when `settings-view` is disabled, describint how to re-enable it
```
Comment (Daeraxa) [inline CHANGELOG.md:21]:
```suggestion
- Added warning when `settings-view` is disabled, describing how to re-enable it
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,21 @@
...
+- Fixed a bug where devtools won't open (https://github.com/pulsar-edit/pulsar/issues/260)
+- Fixed a bug where the editor refused to open with the message "GPU process
+isn't usable. Goodbye" (https://github.com/pulsar-edit/pulsar/issues/233)
+- Fixed logo artifacts on Linux
+- Fixed commands like `--version`, `--package` or `--help` did not show outputs
+- Fixed additional flags not being sent to `--package`
+- Small improvement on the binary size
+- Fixed "install command line tools" on Mac and Windows
+- Cached queries for featured packages (featured packages will load faster, and
+less errors on the settings-view regarding package info)
```
Comment (Daeraxa) [inline CHANGELOG.md:20]:
```suggestion
fewer errors on the settings-view regarding package info)
```

**Comment by Daeraxa** [comment]
Only a small one but about my only recent contribution which was adding `--no-sandbox` to the `yarn start` which solves the same things as @confused-Techie's PRs but on the source version. (https://github.com/pulsar-edit/pulsar/pull/276)

Also some additional rebranding and relinking to our website - https://github.com/pulsar-edit/pulsar/pull/282

**Comment by confused-Techie** [comment]
> Only a small one but about my only recent contribution which was adding `--no-sandbox` to the `yarn start` which solves the same things as @confused-Techie's PRs but on the source version. (#276)
> 
> Also some additional rebranding and relinking to our website - #282

I think this might just be adding the bigger things, since it doesn't seem to be the big giant ChangeLog like is below it, so we could always add those things to the more in depth changeLog below it if not here

**Comment by mauricioszabo** [comment]
@Daeraxa I just added the "macro" things that we did, so that the change log doesn't get too massive to read.

**Review by confused-Techie — APPROVED** [review]
LGTM!

**Pull request merged**